app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -14,8 +14,7 @@ class GetUspsProofingResultsJob < ApplicationJob
   queue_as :long_running
 
   def perform(_now)
-    return true unless ipp_enabled?
-    return true if ipp_ready_job_enabled?
+    return true unless job_can_run?
```

Comment:
IMO having a return value in a background job is not useful because nothing can read it? I'd just have this be plain `return`?
```suggestion
    return unless job_can_run?
```

Reply:
👍 fixed in 183db1a7008a765f271a41b31bec756750ce7187